components/site/src/lib.rs

```diff
@@ -583,11 +584,29 @@ impl Site {
         imageproc.do_process()
     }
 
-    /// Deletes the `public` directory if it exists
+    /// Deletes the contents in the `public` directory if it exists
```

Comment:
The behaviour should be the same if the option is not set, the folder should just be removed entirely